examples/apex/apex_policy_engine_active_thread_count.cpp

```diff
@@ -57,35 +55,29 @@ std::uint64_t fibonacci(std::uint64_t n)
 
 using hpx::naming::id_type;
 using hpx::performance_counters::get_counter;
-using hpx::performance_counters::stubs::performance_counter;
+using hpx::performance_counters::performance_counter;
 using hpx::performance_counters::counter_value;
 using hpx::performance_counters::status_is_valid;
 static bool counters_initialized = false;
 static const char * counter_name = "/threadqueue{{locality#{}/total}}/length";
 
-id_type get_counter_id() {
+performance_counter get_counter()
+{
     // Resolve the GID of the performances counter using it's symbolic name.
     std::uint32_t const prefix = hpx::get_locality_id();
-    /*
-    id_type id = get_counter(hpx::util::format(
-        "/threads{{locality#{}/total}}/count/instantaneous/active",
-        prefix));
-    */
-    id_type id = get_counter(hpx::util::format(counter_name, prefix));
-    return id;
+    return performance_counter(hpx::util::format(counter_name, prefix));
 }
 
 void setup_counters() {
     try {
-        id_type id = get_counter_id();
+        performance_counter counter = get_counter();
         // We need to explicitly start all counters before we can use them. For
         // certain counters this could be a no-op, in which case start will
         // return 'false'.
         performance_counter::start(hpx::launch::sync, id);
```

Comment:
```suggestion
        counter.start(hpx::launch::sync, id);
```
I think?

Reply:
Right... I missed that spot (I can't compile it here on my machine), thanks.